.github/workflows/builds-review.yaml

```diff
@@ -0,0 +1,62 @@
+name: Review builds
+
+on:
+  pull_request:
+    types:
+      - labeled
+
+jobs:
+  build:
+    if: |
+      github.event_name == 'pull_request' &&
+      github.event.label.name == 'build::review'
```

Comment:
Before we forget!

```suggestion
      github.event.label.name == 'build::canary'
```

Reply:
Oh right, so I was wondering if we should define canary builds as those that are created when a branch is merged into the main branch, and "review build" if we want to review a pull request. That's why I thought we should use "build::review" for that workflow. Too convoluted?

Reply:
Ah I see, fair enough for me, makes it easier to outline in our docs with clear terminology like this.